apps/search/js/providers/suggestions.js

```diff
@@ -46,6 +46,10 @@
 
     remote: true,
 
+    clear: function() {
+      this.container.innerHTML = '';
+    },
```

Comment:
What's this doing here? Doesn't this get inherited from the provider base class?